www/js/lib/zimArchiveLoader.js

```diff
@@ -70,12 +71,12 @@ define(['zimArchive', 'jquery', 'uiUtil'],
         });
         jQuery.when.apply(null, promises).then(function() {
             callbackFunction(directories);
-        }, function(error) {
-            uiUtil.systemAlert("Error scanning your device storage : " + error
+        }).catch(function (error) {
```

Comment:
Here, can we say that this `callbackError` will be called only in the case when no archives are found? If yes, it can help us write clear function descriptions and comments